mappings/net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket.mapping

```diff
@@ -1,3 +1,8 @@
 CLASS net/minecraft/class_2744 net/minecraft/network/packet/s2c/play/EntityEquipmentUpdateS2CPacket
 	FIELD field_12565 id I
+	FIELD field_25721 equipment Ljava/util/List;
```

Comment:
```suggestion
	FIELD field_25721 equipments Ljava/util/List;
```

Reply:
I am pretty sure ``equipment`` is both plural and singular, as in ``I have some equipment in the car.``.

Reply:
Hmm, I think we need a way to distinguish that this is a list than a single equipment.

Reply:
Maybe ``equipmentList``, or ``equipmentSlotToItem`` (bit verbose)?

Reply:
Or ``equipmentUpdates``?

Reply:
`equipmentList` I suppose

Reply:
Done!